### PR TITLE
Dev mingjun

### PR DESCRIPTION
**Pull request by MingjunWu**
my plugin

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (c-sachdeva) [inline .idea/misc.xml]: change encoding

**Review by c-sachdeva — APPROVED** [review]
Looks cool

File: src/main/java/edu/cmu/cs/cs214/rec10/StartGame.java [hunk 2]
```diff
@@ -10,9 +10,9 @@
 import java.util.List;
 import java.util.ServiceLoader;
 
-public class Main {
+public class StartGame {
```
Comment (c-sachdeva) [inline src/main/java/edu/cmu/cs/cs214/rec10/StartGame.java]: change name

**Review by c-sachdeva — CHANGES_REQUESTED** [review]
change file name

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+
+import java.util.Random;
+
+public class SortWithSwapsPlugin implements GamePlugin {
+    private int ONE = 1;
+    private int TEN = 10;
+
+    // Describes whether the current move is over
+    public Boolean move = false;
```
Comment (AlintaLu) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: Consider make it private; Use boolean instead of Boolean

**Pull request merged**